hera_mc/geo_handling.py

```diff
@@ -168,19 +177,19 @@ def get_all_locations(args):
     return stations_new
 
 
-def get_since_date(args):
-    dt = cm_utils._get_datetime(args.since_date, '0')
+def get_since_date(args,query_date):
+    dt = query_date.gps
     db = mc.connect_to_mc_db(args)
     found_stations = []
     with db.sessionmaker() as session:
-        for a in session.query(geo_location.GeoLocation).filter(geo_location.GeoLocation.created_date >= dt):
+        for a in session.query(geo_location.GeoLocation).filter(geo_location.GeoLocation.created_gpstime >= dt):
             found_stations.append(a.station_name)
     return found_stations
 
 coord = {'E': 'easting', 'N': 'northing', 'Z': 'elevation'}
 
 
-def plot_stations(args, stations_to_plot, fignm, marker_color='g', marker_shape='o', marker_size='8', label_station=False):
+def plot_stations(args, stations_to_plot, fignm, query_date=False, marker_color='g', marker_shape='o', marker_size='8', label_station=False):
```

Comment:
I think you should use None for an undefined query date rather than False so you don't end up with mixed types.